Sources/CustomDump/Conformances/Foundation.swift

```diff
@@ -325,3 +325,9 @@ extension UUID: CustomDumpStringConvertible {
     "UUID(\(self.uuidString))"
   }
 }
+
+extension AnyHashable: CustomDumpRepresentable {
```

Comment:
Minor, but can you move this to [Conformances/Swift.swift](https://github.com/pointfreeco/swift-custom-dump/blob/819d9d370cd721c9d87671e29d947279292e4541/Sources/CustomDump/Conformances/Swift.swift). This file is meant for Foundation framework types.

Reply:
I moved it and added tests for everything that was there already. I also noticed that `ObjectIdentifier` dump was also including the memory address which makes no sense so I changed to remove the memory address from the dump https://github.com/pointfreeco/swift-custom-dump/pull/65